src/window.rs

```diff
@@ -29,8 +29,13 @@ use crate::platform_impl::{self, PlatformSpecificWindowAttributes};
 ///
 /// ## Platform-specific
 ///
-/// **Web:** The [`Window`], which is represented by a `HTMLElementCanvas`, can
-/// not be closed by dropping the [`Window`].
+/// - **Web:** The [`Window`], which is represented by a `HTMLElementCanvas`, can not be closed by
+///   dropping the [`Window`].
+/// - **macOS:** The window can only be created after the application has finished launching. This
+///   is the reason why [`EventLoop`] doesn't contain any functionality to create the window.
+/// - **iOS:** Same as macOS.
```

Comment:
I'm unsure if this is the right place to put it, but I wanted the _motivation_ for the current design to be available somewhere.

Reply:
It sounds like the whole design is motivated solely by macOS, where in reality it's not and window creation being inside the event loop is a bit more complex.

I'd generally just drop it, since it's more confusing then helping, the issue about window design, etc, should provide a _good_ enough reasoning.

Reply:
Fair enough, I've removed it again